### DISCUSSION TITLE
Using sanity with cloudflare pages

### DISCUSSION THREAD

**Discussion by jcheese1** [sanity-io/sanity#3401] [Sanity Studio Support]
Hi. I currently have a project using Remix + Sanity v3 on Cloudflare pages.

However, it seems like having Sanity in the dependency triggers this error when building for prod in cloudflare console: 

```
workers.api.error.script_too_large
```

I'm using the `Studio` component exported from the sanity package...

Removing sanity from the dependency deploys as expected. Is sanity doing something in global scope that takes a while to run? Thanks!

**Comment by diogoribeirodev** [comment]
same problem with me, but with Nextjs
